src/test/java/statisticker/StatisticsTest.java

```diff
@@ -8,31 +8,29 @@
 
 import org.junit.Test;
 
-public class StatisticsTest 
-{
+public class StatisticsTest {
+
     @Test
-    public void reportsAverageMinMaxx()
-    {
+    public void reportsAverageMinMaxx() {
         Float[] numbers = {1.5f, 8.9f, 3.2f, 4.5f};
-        List<___> numberList = Arrays.asList(numbers);
+        List<Float> numberList = Arrays.asList(numbers);
 
         Statistics.Stats s = Statistics.getStatistics(numberList);
 
         float epsilon = 0.001f;
-        assertEquals(s.average, 4.525f, epsilon);
-        assertEquals(s.min, 1.5f, epsilon);
-        assertEquals(s.max, 8.9f, epsilon);
+        assertEquals(4.525f, s.average, epsilon);
```

Comment:
What is the reason to interchange the value and the expected here? The original code has `s.average` first, then `4.525`

Reply:
This interchange is the right thing to do! Nice attention to detail 👍